lib/tasks/test_vmdb.rake

```diff
@@ -1,7 +1,9 @@
 require_relative "./evm_test_helper"
-require 'parallel_tests/tasks'
 
 if defined?(RSpec) && defined?(RSpec::Core::RakeTask)
+
+require 'parallel_tests/tasks'
+
```

Comment:
Rubocop doesn't complain about the lack of indentation for this `if` here and 👇  WAT

Reply:
I'm probably the animal who didn't indent it.

Reply:
Yeah a bunch of these tasks are like that, maybe better to fix them all in a separate PR

Reply:
I won't say who but you're off the hook @jrafanie ;)

Reply:
https://github.com/ManageIQ/manageiq/pull/22832

Reply:
It was added to avoid an indentation commit that modifies every line to prevent churn (not saying I agree, but I remember it happening).  There are a few others like that, such as: https://github.com/ManageIQ/manageiq-appliance_console/blob/c243c0290e8859a5f74580059186e9329ed7fc7d/bin/appliance_console#L79-L80